mw-webapp/src/component/title/Title.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import {Heading} from "@radix-ui/themes";
+import styles from "./Title.module.scss";
+
+export enum HeadingLevel {
+  H1 = "h1",
+  H2 = "h2",
+  H3 = "h3",
+  H4 = "h4",
+  H5 = "h5",
+  H6 = "h6",
+}
+
+interface TitleProps {
+  /**
+   * Heading level
+   */
+  level: HeadingLevel;
+  /**
+   * CSS class name for styling the component.
+   */
+  className: string;
+  /**
+   * Text content of the heading
```

Comment:
`Text content of the heading` -> Title.
text (string) - obviously from trypescript
content - it's not a real content, just string
heading - obviously from context

Reply:
"The title text for the heading" be good?

Reply:
You dont even need to mention heading in the documentation of heading props) the same for some nested properties like Car.CarColor look weird, right? Car.color is good enough. 
But it is really minor thing in current case, I propose to use just word 'Title' but it is up to you)